dev-docs/auth.md

```diff
@@ -15,15 +15,28 @@ We added our own [authn-service](https://github.com/chef/authn-service) as glue
 
 For further questions, the team can be reached at [#auth-team](https://chefio.slack.com/messages/C61F9HHKK/)
 
-## OpenID Connect aka Dex
+## Supported Identity Management Systems
 
-If you want to use google as an IdP (Identity Provider) or our internal test LDAP server, there are a few variables you can set.
+OKTA with SAML and AD with LDAP are the most common configurations among our users.
```

Comment:
`AD with LDAP are the most common configurations among our users.` This is a bit misleading since most people using AD are using the AD specific configuration instead of LDAP. I understand what this is saying but the world `configurations` might be a bit overloaded here.

Reply:
Oh just also realized this was a dev doc. That's an ignorable nit in that case.